tests/basis/basis.js

```diff
@@ -162,6 +162,22 @@ describe('Basis', function () {
         expect(basis.standardRepresentation).to.be.deep.almost.equal(Na4Cl4.basis);
     });
 
+    //** Minimum Lattice Size Molecule//
+    it('should return minimum lattice size for a molecule', function() {
+        const basis = new Basis(C2H4.basis);
+        const minimumBasisSize = 3.162;
+        const latticeSize = basis.getMinimumLatticeSize();
+        expect(latticeSize).to.be.equal(minimumBasisSize);
+    })
+
+    //** Minimum Lattice Size Atom//
+    it('should return minimum lattice size for an atom', function() {
+        const basis = new Basis(Na.basis);
+        const minimumBasisSize = 1.9;
```

Comment:
What is "minimumBasisSize" - is this a minimum pairwise distance for C2H4 and atomic radius for Na?? If yes, we should call them accordingly for clarity or add commentaries